performant-software/neatline-spa
# Record selection and map display fixes

### What does this PR do?
This PR follows up on recent changes and refactors to the UI to the action reducer / saga approach, addressing various bugs relating to record selection, UI state, and map initialization.

### What issues does it address?
- Closes #126 
- Closes #129 

### How to test?
Make a new exhibit and create a first record. Upon successfully saving the record, the UI should show the record's title in the breadcrumb navigation. Using the leaflet draw tools on the map, add some geometry features for the new record. It should save these correctly without any unexpected side effects (e.g. geometry disappearing or a second record being created in the background).
